.github/workflows/release-asset-audit.yml

```diff
@@ -22,7 +22,12 @@ jobs:
     runs-on: ubuntu-24.04
     if: github.repository == 'llvm/llvm-project'
     steps:
-      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 #v4.1.6
+      - name: Checkout LLVM
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
Do we use one specific version in all the workflows? Even if we intended to, I expect some have drifted by now anyway.

(fixing the version to something is good for traceability, no objection to that)